config/settings.yml

```diff
@@ -39,3 +39,6 @@ user:
   locations:
     spec:
       - an.ip.address
+
+cors:
+  allow_origin_url: 'https://embed-dev.stanford.edu'
```

Comment:
Can we make this be just `embed.stanford.edu` as the default behavior (and we'll add exceptions to the appropriate settings files when necessary)?

It seems like the other things services (like PURL and ImageServer) all have their production URLs in this file, so it falls in line with the pattern that's already established.